### DISCUSSION TITLE
how to restore from `coordinator_backup.json`?

### DISCUSSION THREAD

**Discussion by wspeth** [Koenkk/zigbee2mqtt#8112] [General]
important question regarding "coordinator backup": what would zigbee2mqtt employ as a trigger to restore stick from `coordinator_backup.json`?
I want to edit the json by hand to add devices that were paired on a different system (not online same time), having same channel, PAN-ID of course (a cloned system). Then how would I make zigbee2mqtt restore from edited coordinator_backup.json and find the added device paired?

**Comment by Koenkk** [comment]
A restore can be triggered when starting z2m with a freshly (re)flashed stick

**Comment by wspeth** [comment]
thank you for the quick answer. Yet..hm..this does not help solve my task. I would need a means then to erase the stick by some routine I can start by code.
